### PR TITLE
[Patterns]: Update pattern category descriptions

### PR DESCRIPTION
**Pull request by ntsekouras**
## What?
Extracted from: https://github.com/WordPress/gutenberg/pull/45548

>Implements category descriptions suggested by @beafialho in https://github.com/WordPress/gutenberg/issues/44501#issuecomment-1302380989, as a starting point

### CODE REVIEW

File: lib/compat/wordpress-6.2/block-patterns.php [hunk 1]
```diff
@@ -18,50 +18,57 @@ function gutenberg_register_core_block_patterns_and_categories() {
...
+			'label'       => _x( 'Buttons', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Patterns that contain buttons and call to actions.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'columns',
 		array(
-			'label' => _x( 'Columns', 'Block pattern category', 'gutenberg' ),
+			'label'       => _x( 'Columns', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Multi column patterns with more complex layouts.', 'gutenberg' ),
```
Comment (Mamaduka) [inline lib/compat/wordpress-6.2/block-patterns.php]: Should we use a hyphen for creating a compound word here - "Multi-column"?

Sorry, I can't remember the exact grammar rule.

**Review by Mamaduka — APPROVED** [review]
This looks good to me, a straightforward change.

I left one note regarding the text copy, where I would appreciate a second opinion.

File: lib/compat/wordpress-6.2/block-patterns.php (hunk 1 above)
Comment (jasmussen) [reply lib/compat/wordpress-6.2/block-patterns.php]: Sounds right to me with the hyphen.

Comment (ntsekouras) [reply lib/compat/wordpress-6.2/block-patterns.php]: Thanks! I updated it.

**Pull request merged**